packages/admin-panel/src/pages/resources/PermissionsPage.js

```diff
@@ -103,8 +103,15 @@ const CREATE_CONFIG = {
   },
 };
 
-// Return an array of records for bulk editing on the server
-const processDataForSave = fieldsToSave => {
+// When creating, return an array of records for bulk editing on the server
+// When editing, just process a single record as normal
+const processDataForSave = (fieldsToSave, recordData) => {
```

Comment:
Would recordData ever be undefined?

Reply:
Good question - it shouldn't be, at the point we're processing data for save.